### PR TITLE
HTCONDOR-2284 fixup recipe example as per RCF feedback

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [x] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/faq/users/prefer-gpu-to-cpu.rst [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-    rank                        = RequestGPUs
-    # If your job requires more than one GPU, you may change both
-    # of the '1's in the next line to some other number.
-    request_GPUs                = countMatches(RequireGPUs, AvailableGPUs) >= 1 ? 1 : 0
+    ## Execution section. You will need to change these for your job
+    executable                  = /bin/sleep
+    arguments                   = 600
 
-    ## Requirements section.
+    ## Requirements section. Also, probably needs customizatoin.
```
Comment (ColeBollig) [inline docs/faq/users/prefer-gpu-to-cpu.rst:14]:
```suggestion
    ## Requirements section. Also, probably needs customization.
```

File: docs/faq/users/prefer-gpu-to-cpu.rst [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
     request_CPUs                = 1
+    request_Memory              = 2G
+    request_Disk                = 20G
     require_GPUs                = Capability > 4
 
     ## File transfer section.
     should_transfer_files       = true
-    transfer_executable         = false
+    transfer_input_files        = input1, input2
```
Comment (ColeBollig) [inline docs/faq/users/prefer-gpu-to-cpu.rst:22]: Should we note to modify or remove `transfer_input_files` in the submit file? Or is it implied?

File: docs/faq/users/prefer-gpu-to-cpu.rst [hunk 3]
```diff
@@ -1,26 +1,32 @@
 Example: Run a Job on a CPU if a GPU is not Available
 =====================================================
 
+Some jobs may be able to take advantage of GPU if one is available, but
+can run on a CPU if there are no slots with GPUs immediately matchable.
+The following submit file will match a job in this case.
```
Comment (aowen-uwmad) [inline docs/faq/users/prefer-gpu-to-cpu.rst:6]: Add link to page explaining what `rank` does?

File: docs/faq/users/prefer-gpu-to-cpu.rst [hunk 4]
```diff
@@ -1,26 +1,32 @@
...
+
 .. code-block:: condor-submit
 
-    ## Magic section: don't change anything in this section unless you
-    ## understand why it works.
-    rank                        = RequestGPUs
-    # If your job requires more than one GPU, you may change both
-    # of the '1's in the next line to some other number.
-    request_GPUs                = countMatches(RequireGPUs, AvailableGPUs) >= 1 ? 1 : 0
+    ## Execution section. You will need to change these for your job
```
Comment (aowen-uwmad) [inline docs/faq/users/prefer-gpu-to-cpu.rst:10]: Add link to the current page, so when user copy/pastes they can come back to this document?

Comment (GregThain) [reply docs/faq/users/prefer-gpu-to-cpu.rst:14]: This is fantastic, thanks!

**Pull request merged**